hh/domain-eng.yml

```diff
@@ -135,9 +135,12 @@ responses:
 
   utter_more_terms:
       - text: |
-          You confirm that you are responsible for your medical care & treatment. This service only provides info.
-          This service is not a substitute for professional medical advice/diagnosis/treatment. Get a qualified health provider's advice about your medical condition/care
-          You confirm that you shouldn't disregard/delay seeking medical advice about treatment/care because of this service. Rely on info at your own risk.
+          _You confirm that you’re responsible for your medical care & treatment.This service only provides info.
+
+          This service is not a substitute for professional medical advice/diagnosis/treatment. Get a qualified health provider's advice about your medical condition/care, especially if you develop severe symptoms.
+
+          You confirm that you shouldn't disregard/delay seeking medical advice about treatment/care because of this service. Rely on info at your own risk._
+      - document: "https://healthcheck-rasa-images.s3.af-south-1.amazonaws.com/NDOH_WhatsApp+Covid-19_ABRIDGED+Privacy+Policy.pdf"
```

Comment:
I think this mustn't have a `-`. You want it to be:
```
"utter_more_terms": [
  {
    "text": "...",
    "document": "..."
  }
]
```
That's what the CI is complaining about